### DISCUSSION TITLE
Missing translation strings

### DISCUSSION THREAD

**Discussion by vmario89** [requarks/wiki#6077] [Error / Bug Report]
Hi,

recently i started to try out Wiki.js and so far it seems to be the best open source Wiki system in the whole web in terms of availibity, ease of setup and use.

I already wanted to participate for localization. As i did not want to create a new issue first, i wanted to open a discussion about the translation status of the application. I recognized there are a lot of strings missing in the language json files. Just one example for a page in the admin backend, which is missing translations strings "Security -> Security":

![grafik](https://user-images.githubusercontent.com/5631071/213925429-32638771-f4df-4f7b-b37e-f35fc9299ba0.png)

by the way when joining the lokalise project, they want me to make an account which seems to need payment after 14 days. Is this intended?

https://app.lokalise.com/signup/2994254859f751ea605a00.03473540/de/
![grafik](https://user-images.githubusercontent.com/5631071/213925768-0d623e0f-3db5-486a-a7e1-843cb2cca72e.png)

**Comment by cdn0x12** [comment]
Currently not every string is localized. Nicolas, the core dev, said on slack that in v3 all strings will be localized.

Since localise tells you no credit card required, you won't be charged unless you bind your card to your account and make a subscription. Participating in others' projects does not require any subscription while creating your own project may do so.

**Comment by vmario89** [comment]
Hi. Thanks for your answer! Both sounds fine. Could you add the last sentence about the lokalise project maybe on your website? I guess there are more people like me, wondering about that cost information and maybe stop to continue joining the localise team.
cheers, Mario
